src/inc/quic_platform_winkernel.h

```diff
@@ -494,58 +494,222 @@ _CxPlatEventWaitWithTimeout(
 #define CxPlatEventWaitWithTimeout(Event, TimeoutMs) \
     (STATUS_SUCCESS == _CxPlatEventWaitWithTimeout(&Event, TimeoutMs))
 
+#ifndef IO_COMPLETION_ALL_ACCESS // TODO: Remove onces APIs are published
```

Comment:
Isn't all this already included from `msquic_winkernel` (which is included in this file?)